rpcs3/Emu/System.cpp

```diff
@@ -674,19 +694,8 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		// Load patches from different locations
 		g_fxo->get<patch_engine>().append_title_patches(m_title_id);
 
-		// Mount all devices
-		const std::string emu_dir = rpcs3::utils::get_emu_dir();
 		std::string bdvd_dir = g_cfg.vfs.dev_bdvd;
 
-		// Mount default relative path to non-existent directory
```

Comment:
why exactly is this code moved?
isn't this already done correctly here after only global configs are loaded?

Reply:
nevermind. I mixed it up with the stuff in Init

Reply:
wait. if you restore the settings, then this move shouldn't be necessary right?

Reply:
I believe no change in System.cpp in this PR is necessary. This is cutting off the possibility to configure custom paths for a game by manually editing custom config, for the sake of automatically hammer-fixing some supposedly broken configs.

Reply:
Done.